src/main/java/seedu/address/model/project/Deadline.java

```diff
@@ -31,6 +32,7 @@ public class Deadline {
      */
     public Deadline(String deadline) {
```

Comment:
I think we should change the value stored in the `Deadline` to a `LocalDate` or `LocalDateTime` object so that in future if we decide to change the expected date format, we only need to change the parsing function. We could add this as a code quality improvement in v1.4

Reply:
Optional<LocalDate> would be the most optimal, since we need a value for "no deadline set" as well.

Reply:
Perhaps a better design would be `Optional<Deadline>` in a project, and a `LocalDate` in `Deadline`? Since a project **can** have a `Deadline`, but a `Deadline` **must** have a `LocalDate`

Reply:
Yes, I think that would be the most optimal indeed.